romancal/pipeline/exposure_pipeline.py

```diff
@@ -117,35 +117,38 @@ def process(self, input):
             if input_filename:
                 result.meta.filename = input_filename
             result = self.saturation(result)
+            # pdb.set_trace()
 
             # Test for fully saturated data
             if is_fully_saturated(result):
                 log.info("All pixels are saturated. Returning a zeroed-out image.")
 
+                if "groupdq" in result.keys():
```

Comment:
Why is this dependent on whether `groupdq` is present? Isn't it sufficient to check if the result is saturated?

Reply:
You are correct and the groupdq check is removed.

Reply:
Oh, I see - the check is probably because the `[is_saturated](https://github.com/spacetelescope/romancal/blob/b6f7cb6dd3eb4258c30db95a1d08f4c58dea054c/romancal/lib/basic_utils.py#L47)` function checks for saturation in the groupdq array first and it will crash if not available. I'm not sure if we'll get data without a groupdq array and even if we do the check probably needs to be handled in the `is_saturated` function.

Reply:
Approved but see comment above.

Reply:
I think leaving the groupdq out here is ok. You are correct that is_saturated needs the groupdq and the utility can be passed any file so it needs to check. 
For the pipeline we always run dq_init and so when it outputs the ramp model it will have the groupdq and if not it will fail validation. I can add it back if you think it is necessary but I don't think it is needed here.